download.md

```diff
@@ -42,30 +40,29 @@ As of 2018-07-02, PELUX 2.0 has been released! The docs can be found
 
 ### Artifacts
 * These release notes.
-* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v2.0/rocko/pelux.xml)
+* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v3.0/sumo/pelux.xml)
   in PELUX Yocto Manifests repository, see below.
 * The version of [PELUX Software Development
-  Environment](https://github.com/Pelagicore/pelux-sde/tree/v2.0) as pointed out
-  below.
-* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/PELUX-2.0/) as
+  Environment](https://github.com/Pelagicore/pelux-sde/tree/v3.0) as pointed out below.
+* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/PELUX-3.0/) as
   pointed out below.
```

Comment:
Probably its a good idea to add links here such as:

- [Qemu Artifacts](https://pelux.io/artifacts/archive/artifacts_qemu-x86-64_nogfx/) 
- [Intel minimal images](https://pelux.io/artifacts/archive/artifacts_intel/) 

And so on.. But maybe its best to wait until these are actually ready.

Reply:
@fhajredini Have your changes already been done? Shouldn't the addition of the links you mention be done after you are finished?